pkg/reconciler/cosigned/clusterimagepolicy_internal.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 The Sigstore Authors.
```

Comment:
Suggestion: If this version is internal, it could go under `pkg/api/cosigned` where the rest of version are located. I prefer that rather than in the reconciler.

Reply:
Agreed, I just pushed this the moment I got it working to get early reviews before more potential changes

Reply:
Hmm, for now, I moved it to `pkg/api/cosigned/private`
I couldn't use `pkg/api/cosigned/internal`
Reason explained here I believe:
https://stackoverflow.com/questions/59342373/use-of-internal-package-not-allowed